cumulusci/tasks/salesforce/enable_prediction.py

```diff
@@ -0,0 +1,49 @@
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+from cumulusci.core.exceptions import SalesforceException
+from simple_salesforce.exceptions import SalesforceError
+
+
+class EnablePrediction(BaseSalesforceApiTask):
+    task_docs = """
+    cci task run enable_prediction --org dev -o developer_name mlpd__Example_Prediction_v0
+    """
+
+    task_options = {
+        "developer_name": {
+            "description": "Developer name of the MLPredictionDefinition.",
+            "required": True,
+        }
+    }
+
+    def _run_task(self):
+        self._enable_prediction()
+
+    def _enable_prediction(self):
+        try:
+            ml_prediction_definition_id = self._get_ml_prediction_definition_id()
+
+            metadata = self.tooling._call_salesforce(
+                method="GET",
+                url=f"{self.tooling.base_url}sobjects/MLPredictionDefinition/{ml_prediction_definition_id}",
+            ).json()["Metadata"]
+
+            metadata["status"] = "Enabled"
```

Comment:
Should we make it possible to also change to statuses Draft and Disabled?

Reply:
It's not necessary for enabling the prediction, so I invoke the YAGNI principle here.